src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -442,8 +442,8 @@ public static int execute(final String[] args) throws Exception {
             ksAlias = line.getOptionValue("ksalias", ksAlias);
             index = line.getOptionValue("i", index);
 
-            String enabledCiphersString = line.getOptionValue("ec", null);
-            String enabledProtocolsString = line.getOptionValue("ep", null);
+            String enabledCiphersString = line.getOptionValue("ec",  (String) null);
```

Comment:
Why not just `line.getOptionValue("ec")`?

The docs show a method that doesn't have a defaultValue as a second param: https://commons.apache.org/proper/commons-cli/apidocs/org/apache/commons/cli/CommandLine.html#getOptionValue(char)

Reply:
Sure but AFAIU any user can set `ec` and `ep` to the empty string. and all code is relaying on `null` values. So I decided to leave it as.